### PR TITLE
Add demo files for EuroBioC2023

### PR DESCRIPTION
**Pull request by TuomasBorman**
This includes files for demonstration that will be kept in EuroBioC2023 conference. Some work to be done still

### CODE REVIEW

File: demo/EuroBioC2023/workflow.Rmd [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+y <- "Count"
+
+plot <- ggplot(df, aes(x = .data[[x]], y = .data[[y]])) +
+    geom_bar(stat="identity")
+plot
+```
+
+### Big data
+
+- Add here something about cohort studies, big data, how analysis is done with big data, requirements...
```
Comment (TuomasBorman) [inline demo/EuroBioC2023/workflow.Rmd]: Add text

File: demo/EuroBioC2023/workflow.Rmd [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+is a EMBL-EBI's database for metagenomic data. This large microbiome database
+can be accessed with _MGnifyR_ package which nowadays support _TreeSE_. The
+package will be submitted to Bioconductor's next release.
+
+We chose this dataset...
+
+As loading takes some time, the dataset is already loaded.
+
+For other available datasets and importing methods, check [OMA](https://microbiome.github.io/OMA/containers.html#example-data).
+
```
Comment (TuomasBorman) [inline demo/EuroBioC2023/workflow.Rmd]: Create a workflow e..g. with mia dataset. --> Let's switch the dataset later.

1. Summarize the data
2. Subset the data
3. Create plots with miaViz
4. Perform dbRDA
5. Some basic DAA

File: demo/EuroBioC2023/workflow.Rmd [hunk 3]
```diff
@@ -0,0 +1,426 @@
...
+```{r}
+colData(altExp(MAE[[1]], "Species")) %>% as_tibble() %>%
+    tidyr::pivot_longer(cols=c(shannon,observed), names_to = "alpha_type",
+                        values_to = "estimate") %>% 
+    ggplot(aes(x=estimate)) +
+    geom_histogram(color = "black", fill = "gray", bins = 30)+
+    facet_wrap(~alpha_type, nrow = 1, scales = "free")+
+    labs(title = "Alpha Diversity")+
+    theme(plot.title = element_text(hjust = 0.5))
+```
```
Comment (TuomasBorman) [inline demo/EuroBioC2023/workflow.Rmd]: You could compare e.g. shannon between groups --> plotColData()

File: demo/EuroBioC2023/workflow.Rmd [hunk 4]
```diff
@@ -0,0 +1,426 @@
...
+                        }
+                    },
+                heatmap_legend_param = list(title = "", legend_height = unit(5, "cm")),
+                column_names_gp = grid::gpar(fontsize = 8),
+                row_names_gp = grid::gpar(fontsize = 8),
+                column_names_centered=TRUE
+                )
+plot
+```
+
```
Comment (TuomasBorman) [inline demo/EuroBioC2023/workflow.Rmd:447]: How about example on differential abundance analysis using ancombc2

File: demo/EuroBioC2023/workflow.Rmd [hunk 5]
```diff
@@ -0,0 +1,437 @@
...
+- Cohort datasets are large in size
+    - Data management, handling and wrangling --> data structure
+    - Computational power --> High performance computing (HPC) and cloud computing
+
+- [MultiAssayExperiment (MAE)](https://bioconductor.org/packages/release/bioc/html/MultiAssayExperiment.html) and [SummarizedExperiment (SE)](https://bioconductor.org/packages/release/bioc/html/SummarizedExperiment.html)
+    - Several R packages frameworks are increasingly integrating MAE and SE
+    - MAE enables linking of multiple experiments
+    - SE is an efficient data container to store data from an experiment
+
```
Comment (TuomasBorman) [inline demo/EuroBioC2023/workflow.Rmd]: I tied to shorten this little bit. Does it still have the most important info?

**Comment by TuomasBorman** [comment]
@ChouaibB now "demo" file is the final one. Can you check it?

File: demo/EuroBioC2023/workflow.Rmd (hunk 5 above)
Comment (antagomir) [reply demo/EuroBioC2023/workflow.Rmd]: should we highlight that TreeSE has many benefits? (it is SCE, and has metadata slot, feature sequence slot, altExps, trees etc. that SE doesnt have)

File: demo/EuroBioC2023/workflow.Rmd (hunk 4 above)
Comment (antagomir) [reply demo/EuroBioC2023/workflow.Rmd:447]: ok whynot

File: demo/EuroBioC2023/workflow.Rmd (hunk 2 above)
Comment (antagomir) [reply demo/EuroBioC2023/workflow.Rmd]: How about demo on splitByRanks?

**Pull request merged**